patternfly-docs/site/pages/index.js

```diff
@@ -1,12 +1,11 @@
 import React from 'react';
 import { PageSection } from '@patternfly/react-core';
-import quickRefPdf from './reference-docs/PF-quick-ref.pdf';
 
 const HomePage = () => (
   <PageSection variant="light">
-    Just getting started? Take a look at the <a href="/contribution">Contribution guide</a> and{' '}
-    <a href="/guidelines">Coding guidelines.</a> Always keep our <a href="/accessibility-guide">Accessibility guide</a>{' '}
-    in mind. Download a <a href={quickRefPdf}>quick reference sheet.</a>
+    Just getting started? Take a look at the <a href="/contribution">contribution guide</a> and{' '}
+    <a href="/guidelines">coding guidelines</a>, and always keep our <a href="https://www.patternfly.org/accessibility/about-accessibility">accessibility fundamentals</a>{' '}
```

Comment:
One thing - the link is "coding guidelines" but the page title is "Development guidelines" - what do you think about making them match?